modules/setting/setting.go

```diff
@@ -158,6 +158,7 @@ var (
 		TrustedUserCAKeysParsed               []gossh.PublicKey  `ini:"-"`
 		PerWriteTimeout                       time.Duration      `ini:"SSH_PER_WRITE_TIMEOUT"`
 		PerWritePerKbTimeout                  time.Duration      `ini:"SSH_PER_WRITE_PER_KB_TIMEOUT"`
+		RedirectPath                          string             `ini:"SSH_REDIRECT_PATH"`
```

Comment:
This should be mentioned in the docs(under config cheat sheet).

Reply:
Do you think it would be useful for some other interface as well?

Reply:
You can do git operations without SSH. I think it's also nice to have this feature for the HTTP counter-part.

Reply:
Ok, I will have a look at if we can use the redirect / default prefix also for the HTTP interface.

Reply:
Can the documentation be added for this key?

Reply:
yes, I will fix that.